evasquez26/angular4-startup
# S1 one 140 setup integrations

## Story / Task

see [ONE-140](https://bixlabs.atlassian.net/browse/ONE-140)

## What this PR does?

Sets up the inital integration with an external API.


### Before 

- There was no integration with an external API

### After

- We can have an API Url configured for further requests
- We set up an initial endpoint for login
- Webpack configuration had to be exctracted from the project using `ng eject` to be able to set up environmental variables, we can add more later on this file now

### How to test / reproduce

- Execute `API_URL=<yourUrl:port> npm run start>
- Aplication should start up normally

### TODOs

### Related PRs

N/A

## Migrations

NO

Thank you!
